network-api/networkapi/management/commands/fix_local_site_bindings.py

```diff
@@ -1,15 +1,24 @@
 from django.core.management.base import BaseCommand
 from wagtail.models import Site
 
+from networkapi.donate.models import DonateLandingPage
+from networkapi.mozfest.models import MozfestHomepage
+from networkapi.wagtailpages.models import Homepage
+
 
 class Command(BaseCommand):
     help = "Ensure site bindings are for localhost (e.g. for after a db copy)"
 
     def handle(self, *args, **options):
         for site in Site.objects.all():
-            if site.is_default_site:
+            root_page = site.root_page.specific
+
+            if isinstance(root_page, Homepage):
                 site.hostname = "localhost"
-            else:
+            elif isinstance(root_page, MozfestHomepage):
                 site.hostname = "mozfest.localhost"
+            elif isinstance(root_page, DonateLandingPage):
+                site.hostname = "donate.localhost"
+
```

Comment:
Made a small update to this command, which is run after copying the prod or staging DB, and updates the sites domains from either `mofostaging.net` or `foundation.mozilla.org`, back to `localhost`.

The logic of "its either the foundation site or mozfest" is not going to work now that we have added donate as a third site.

What this function is now doing is checking the page type of the site's `root_page`.
 
For example, if the site's root page is of the type `MozfestHomepage` its safe to assume that its host name should be changed to `mozfest.localhost`.

Reply:
Smart!